core/engine.go

```diff
@@ -45,11 +45,16 @@ const (
 	BackoffMax    = 10 * time.Second
```

Comment:
Can you delete TickRate last used in ed8a2462, ShutdownTimeout last used in 2205fe33, BackoffAmount and BackoffMax last used in 371b4af8 and rename all the others to lowercase while you are retouching this :D Maybe a Todo to have them configurable

Reply:
Resolved by https://github.com/loadimpact/k6/pull/1390/commits/f2ca5b08beff01637aeba8d0e8d1ad27bf321374